src/components/header.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Group, Mark, Title } from '@mantine/core';
+import classes from '@/app/global.module.css';
+import Link from 'next/link';
+import { bulkServerURLState } from '@/state/bulk-server-url-state';
+import { useRecoilValue } from 'recoil';
+
+/*
+ * Component for the application header shared across all pages.
```

Comment:
```suggestion
 * Component for the application header to be shared across all pages.
```
maybe? unless the former was intentional but I think this sounds a bit better